### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by Richd0tcom**
Persisting state with local storage

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
   const changeColor = ()=>{
 
-    let hx = (Math.random() * 0xfffff * 100000).toString(16).slice(0,6);
-    setColor(
-      `#${hx}`
-    )
+    let currentColor = `#${(Math.random() * 0xfffff * 100000).toString(16).slice(0,6)}`;
+    setColor(currentColor);
+
+    setHistory([currentColor, ...history].slice(0,6));
```
Comment (johnotu) [inline src/App.js]: @Richd0tcom Something here makes it keep only 5 colors in history rather than the recommended 10.

**Review by johnotu — CHANGES_REQUESTED** [review]
Nice work. See comment about the number of colors in history array.

File: src/App.js (hunk 1 above)
Comment (Richd0tcom) [reply src/App.js]: Yeah I initially made it 5 to test it. I make the corrections as soon as possible

**Review by Richd0tcom — COMMENTED** [review]
okay

**Review by johnotu — APPROVED** [review]

**Pull request merged**